fs.go

```diff
@@ -83,6 +83,47 @@ func (s *System) GetFileSystemByName(name string) (*types.FileSystem, error) {
 	return nil, errors.New("Couldn't find file system")
 }
 
+// GetFileSystemByIDName returns filesystems by name or id
+func (s *System) GetFileSystemByIDName(id string, name string) (*types.FileSystem, error) {
+	defer TimeSpent("GetFileSystemByID", time.Now())
+
+	if id != "" {
```

Comment:
Since ID and name are mutually exclusive parameters, we don't need to have multiple if statements to cover these cases, as giving both of them is not an allowed option. Please club all of them under multiple if-else.

Applicable to other instances of NAS and NFS exports as well.